mobile/src/navigations/onboarding-navigation/components/steps/steps.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+    type DrawerContentComponentProps,
+    DrawerContentScrollView,
+} from '@react-navigation/drawer';
+import React from 'react';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { Step } from '../components';
+import { styles } from './styles';
+
+const Steps = (props: DrawerContentComponentProps): JSX.Element => {
+    const { navigation, state } = props;
+
+    return (
+        <DrawerContentScrollView
+            {...props}
+            style={[globalStyles.m25, globalStyles.flex1]}
+        >
+            <Pressable
+                onPress={(): void => {
+                    navigation.closeDrawer();
+                }}
+                style={styles.button}
+            >
+                <Icon name="close" size={40} color="#D5DCE8" />
+            </Pressable>
+            <Text category="H2" style={styles.title}>
+                Steps
+            </Text>
```

Comment:
I looked at the other code and it seems better to use as:
```
<Text category={TextCategory.H2} style={styles.title}>
      Steps
</Text>
```

Reply:
Yes, you're right, I'll change it tommorow in the morning